server/tests/integration/listings.test.ts

```diff
@@ -127,4 +133,12 @@ describe('Listings endpoints', () => {
       );
```

Comment:
can you also add assert here about mock is never called?

Reply:
@ifhuang I made a big change to my mock implementation (allow choosing between restoring and mocking auth). By default, actual implementation of the middleware is applied. Hence I think there is no need to assert that auth mock is never called anymore, because if it has been called, then response would have been 401, and any of the tests would not have passed.